src/chainparams.h

```diff
@@ -139,13 +145,15 @@ const CChainParams &Params();
 /** Return parameters for the given network. */
 CChainParams &Params(CBaseChainParams::Network network);
```

Comment:
Oh, hmm, this is not defined anywhere anymore (nor is it called anywhere, of course, else we would have compilation errors).